build/gulpfile.vscode.js

```diff
@@ -164,14 +164,13 @@ const config = {
 	productAppName: product.nameLong,
 	companyName: 'Microsoft Corporation',
 	copyright: 'Copyright (C) 2019 Microsoft. All rights reserved',
-	darwinIcon: 'resources/darwin/code.icns',
+	darwinIcon: product.quality === 'stable' ? 'resources/darwin/code.icns' : 'resources/darwin/code-insiders.icns', // {{SQL CARBON EDIT}} Use separate icons for non-stable
```

Comment:
lt's sync with @kburtram offline, because when we move to distros this code will just be reverted anyways, so we might want to wait on this.

Reply:
@anthonydresser - Sorry, didn't read this until now. I synced with Karl and he was fine with this just with the knowledge that it'll be reverted at some point (and changing the .iss changes). Let me know if you still want to discuss this further otherwise I plan on merging this in once I get a validation build with the latest changes and verify things are still working as expected.